### DISCUSSION TITLE
Question about security software warning

### DISCUSSION THREAD

**Discussion by RimacC2-EV** [FoldingAtHome/fah-client-bastet#261] [Q&A]
I just received a warning from my security software (it treats our program as malware), does our FAH software have actions similar to some malware?

**Comment by kbernhagen** [comment]
It downloads signed executable code (the cores).

**Comment by jcoffland** [comment]
Ideally you should file a report with your security software.

As @kbernhagen said, the folding client does download executable software which may be deemed suspicious but a lot of security software is overzealous and it may be triggering on just about anything.

**Comment by RimacC2-EV** [comment]
Thanks for your answer!
(a VERY late reply,sorry 🙁)
________________________________
